pyproject.toml

```diff
@@ -0,0 +1,195 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+#
+# Copyright (C) 2024 Collabora Limited
+# Author: Jeny Sadadia <jeny.sadadia@collabora.com>
+
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "kernelci"
+version = "1.0"
+description = "KernelCI core functions"
+readme = "README.md"
+requires-python = ">=3.8"
+license = {text = "LGPL-2.1-or-later"}
+dependencies = [
+  "azure-storage-file-share==12.13.0",
+  "bson==0.5.10",
+  "click==8.1.3",
+  "cloudevents==1.9.0",
+  "docker==6.1.2",
+  "jinja2==3.1.3",
+  "kubernetes==26.1.0",
+  "paramiko==3.4.0",
+  "pydantic==1.10.5",
+  "pyelftools==0.29",
+  "pytest==7.4.0",
+  "pyyaml==6.0",
+  "requests==2.31.0",
+  "scp==0.14.5",
+  "toml==0.10.2",
+]
+authors = [
+  {name = "Guillaume Tucker" , email = "guillaume.tucker@collabora.com"},
+  {name = "Tyler Baker" , email = "tyler.baker@linaro.org"},
+  {name = "Denys Fedoryshchenko" , email = "denys.f@collabora.com"},
+  {name = "Mark Brown" , email = "broonie@kernel.org"},
+  {name = "Kevin Hilman" , email = "khilman@baylibre.com"},
+  {name = "Matt Hart" , email = "matthew.hart@linaro.org"},
+  {name = "Jeny Sadadia" , email = "jeny.sadadia@collabora.com"},
+  {name = "Quentin Schulz" , email = "quentin.schulz@free-electrons.com"},
```

Comment:
This one could probably be dropped, or updated with a bootlin address.  The free-electrons.com domain doesn't exist anymore (following a trademark dispute with Free ISP iirc around 2020).